addons/vehicles/script_component.hpp

```diff
@@ -0,0 +1,16 @@
+#define COMPONENT Vehicles
+#define COMPONENT_BEAUTIFIED Vehicles
+#define AUTHOR Ashes
+#include "\x\sws\addons\main\script_mod.hpp"
+
+// #define DEBUG_MODE_FULL
+// #define DISABLE_COMPILE_CACHE
+
+#ifdef DEBUG_ENABLED_INGAME
+    #define DEBUG_MODE_FULL
+#endif
+#ifdef DEBUG_SETTINGS_WEAPONS
+    #define DEBUG_SETTINGS DEBUG_SETTINGS_INGAME
+#endif
+
+#include "\x\sws\addons\main\script_macros.hpp"
```

Comment:
```suggestion
#include "\x\sws\addons\main\script_macros.hpp"

#undef AUTHOR
#define AUTHOR Ashes
```

I'm gonna create a better author system in the future but this'll work for now.